simple_history/templates/simple_history/object_history_form.html

```diff
@@ -21,6 +21,14 @@
   {% include "simple_history/submit_line.html" %}
 {% endblock %}
 
+{% block object-tools-items %}
+{# We override this block from the django template to fix up the history link #}
+<li>
+    <a href="{% add_preserved_filters history_url %}" class="historylink">{% translate "History" %}</a>
+</li>
+{% if has_absolute_url %}<li><a href="{{ absolute_url }}" class="viewsitelink">{% translate "View on site" %}</a></li>{% endif %}
```

Comment:
Is there a reason to not use `{% change_form_object_tools %}` from `admin_notify`?

Reply:
I'm not sure what you mean by `admin_notify` I didn't see a django app by that name. 

The `{% change_form_object_tools %}` template tag is the default behavior of the `object-tools-items` block. We can't use it because that template is always overriding the `history_url` context variable to point to the django history version instead of using the simple history version from the context. https://github.com/django/django/blob/main/django/contrib/admin/templates/admin/change_form_object_tools.html#L4

It's definitely not ideal to have to implement all of the extra pieces that would normally be handled by that template tag.

Reply:
Ah, my bad. I didn't go deep enough.